docs/groovy-script/mods/selectionguicrafting/sgc_recipe.md

```diff
@@ -0,0 +1,249 @@
+---
+
+# Crafting Recipe (Selection GUI Crafting)
+
+## Description
+
+Creates a recipe that is shown in the specified category. It can requires an input, output and tool. You may also need a catalyst to craft the recipe.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.selectionguicrafting.sgc_recipe/* Used as page default */ // [!code focus]
+mods.selectionguicrafting.sgcrecipe
+mods.selectionguicrafting.sgcRecipe
+mods.selectionguicrafting.SgcRecipe
+```
+
+
+## Adding Recipes
+
+### Recipe Builder
+
+Just like other recipe types, the Crafting Recipe also uses a recipe builder.
+
+Don't know what a builder is? Check [the builder info page](../../getting_started/builder.md) out.
+
+:::::::::: details mods.selectionguicrafting.sgc_recipe.newRecipe() {open id="abstract"}
+- `Integer`. The amount of XP that will be granted to the player when the recipe is crafted. Requires greater than or equal to 0. (Default `0`).
+
+    ```groovy:no-line-numbers
+    xp(int)
+    ```
+
+- `Integer`. The time in ticks it takes to craft the recipe. Requires greater than or equal to 0. (Default `0`).
+
+    ```groovy:no-line-numbers
+    time(int)
+    ```
+
+- `ArrayList<GsTool>`. The tool that will be used to craft the recipe. Tool refers to the mainhand item. This doesn't need to be an item with durability. Requires greater than or equal to 1.
+
+    ```groovy:no-line-numbers
+    tool(IIngredient)
+    tool(IIngredient, float)
+    tool(IIngredient, float, float)
+    ```
+
+- `ResourceLocation`. The path to the frame texture. Overrides the category frame.
+
+    ```groovy:no-line-numbers
+    frame(String)
+    ```
+
+- `ArrayList<Ingredient>`. The input items that are required to craft the recipe. Input refers to the offhand item. Requires greater than or equal to 1.
+
+    ```groovy:no-line-numbers
+    input(IIngredient)
+    ```
+
+- `ArrayList<GsOutput>`. The output items that will be given to the player. You can add multiple outputs each with a different chance. Requires greater than or equal to 1.
+
+    ```groovy:no-line-numbers
+    output(IIngredient)
+    output(IIngredient, float)
+    ```
+
+- `ArrayList<GsSound>`. The sounds that will be played when the recipe is crafted. Overrides the category sounds.
+
+    ```groovy:no-line-numbers
+    sound(String, float, float)
+    ```
+
+- `GsCatalyst`. The catalyst that will be used to craft the recipe. The catalyst is an optional additional input that must be present in the player's inventory. You're able to set the chance of the catalyst being consumed. Requires greater than or equal to 0 and less than or equal to 1.
+
+    ```groovy:no-line-numbers
+    catalyst(IIngredient)
+    catalyst(IIngredient, float)
+    ```
+
+- `String`. The category that the recipe belongs to. Requires not null.
```

Comment:
doesnt it also require an existing category? or is a category not existing fine here.